### ISSUE TITLE
⚠️ release v0.85.0 build output

### ISSUE THREAD

**Issue by zombieprocess** [MRN-Code/blockly#5]
This was output in a `releaseToProduction` but the release process did not error-out and continued like normal. Just documenting this as a potential problem.

```
>> /coins/coins_builder/src/master/blockly/Gruntfile.js:

>> Exception in thread Thread-2:
>> Traceback (most recent call last):
>>   File "/usr/lib/python2.7/threading.py", line 810, in _bootstrap_inner
>>     self.run()
>>   File "build.py", line 162, in run
>>     self.gen_core()
>>   File "build.py", line 193, in gen_core
>>     self.do_compile(params, target_filename, filenames, "")
>>   File "build.py", line 258, in do_compile
>>     json_data = json.loads(json_str)
>>   File "/usr/lib/python2.7/json/__init_.py", line 338, in loads
>>     return _default_decoder.decode(s)
>>   File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
>>     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
>>   File "/usr/lib/python2.7/json/decoder.py", line 384, in raw_decode
>>     raise ValueError("No JSON object could be decoded")
>> ValueError: No JSON object could be decoded
SUCCESS: msg/js/en.js
SUCCESS: blockly_uncompressed.js
```

**Comment by jwtlake** [comment]
This seems similar 
https://github.com/google/blockly/issues/1399

**Comment by zombieprocess** [comment]
Resolved in #6
